cluster/gce/util.sh

```diff
@@ -3220,6 +3230,59 @@ function create-heapster-node() {
       --metadata-from-file "$(get-node-instance-metadata-from-file)"
 }
 
+# Assumes:
+# - NODE_INSTANCE_PREFIX
+# - PROJECT
+# - NETWORK_PROJECT
+# - REGION
+# - ZONE
+# - ADDITIONAL_MACHINE_TYPE
+# - NUM_ADDITIONAL_NODES
+# - NODE_DISK_TYPE
+# - NODE_DISK_SIZE
+# - NODE_IMAGE_PROJECT
+# - NODE_IMAGE
+# - NODE_SERVICE_ACCOUNT
+# - NODE_TAG
+# - NETWORK
+# - ENABLE_IP_ALIASES
+# - SUBNETWORK
+# - IP_ALIAS_SIZE
+function create-additional-nodes() {
+  local gcloud="gcloud"
+
+  local network=$(make-gcloud-network-argument \
+      "${NETWORK_PROJECT}" \
+      "${REGION}" \
+      "${NETWORK}" \
+      "${SUBNETWORK:-}" \
+      "" \
+      "${ENABLE_IP_ALIASES:-}" \
+      "${IP_ALIAS_SIZE:-}")
+
+  for ((i=1; i<=${NUM_ADDITIONAL_NODES}; i++)); do
```

Comment:
An alternative would be to create a new instance-template from the already existing one and override only machine type and then create a MIG, but  I'm not sure if it's possible to create an instance-template from another one.
Another advantage is that MIG will scale better if someone wants to have a big number of additional nodes (e.g. 500).  

But overall, this looks good to me, it should work for our use case, so it's a good start. We can improve in future.

Reply:
I actually think that we should create MIGs.

Also, I would really like to avoid such huge code duplication. Can't we reuse: "create-nodes-template" and "create-*-nodes" functions?

Reply:
Rewritten to use MIGs